docs/users.rst

```diff
@@ -493,7 +497,7 @@ Yum repository URL
 
 As part of a build request, you may provide the ``repo-url`` parameter with the
 URL to a yum repository file. This file is injected into the container build.
-Note that "ODCS compose" method is disabled if this parameter is given.
+Note that it will be used together with "ODCS compose" method.
```

Comment:
Can we rephrase this or maybe ommit the sentence?

Reply:
I want to mention it, that it can now be used together, rephrased a bit

Reply:
Oh, I see. What would you think about something like

> Since aromic-reactor x.y.z, it is possible to combine ODCS composes with  repository files. In previous versions, the ODCS compose would be disabled if a repository file URL was given.

Then it would be clear that this was not supported before, and it is supported now. If you don't want to mention atomic-reactor in the user docs, it could also go in the lines of 

> Current OSBS versions support the combination of ODCS composes with  repository files. This is a change to OSBS former behavior, where the ODCS compose would be disabled if a repository file URL was given.